types/xmpp__starttls/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-useless-files": false
+    }
```

Comment:
I saw that on Discord. Can this be inline? Otherwise it's hard to see which file is covered by this one

Reply:
You can't use the `// tslint:disable-next-line:[rule]` directive for this (there is no line to disable) and you can't use the `// tslint:disable:[rule]` directive because these directives are forbidden by yet another rule. So I don't see another solution but I'm open to suggestions.

Reply:
which file is this? (I'd checkout PR) Doubt I could help, just want to see this use case you've described on DS

Reply:
It's the empty `index.d.ts` in this package and the empty `index.d.ts` in the `xmpp_iq` package.

Reply:
oh, I see, the only way around this DT convention would be within `index.d.ts` and dropping `starttls.d.ts` file:
```ts
declare module '@xmpp/starttls/starttls' {
    import { Socket } from 'net';
    import { ConnectionOptions, TLSSocket } from 'tls';

    export function canUpgrade(socket: Socket): boolean;
    export function upgrade(socket: Socket, options?: ConnectionOptions): TLSSocket;
}
```
which sounds counterproductive.